monkestation/code/datums/brain_damage/magic.dm

```diff
@@ -43,6 +43,7 @@
 	// Not even nullspace will keep it at bay.
 	for (var/obj/effect/client_image_holder/stalker_phantom/stalker in stalkers)
```

Comment:
Can you set these to be ``as anything`` checks? There shouldn't be unrelated objects within the stalker list. 
```suggestion
	for (var/obj/effect/client_image_holder/stalker_phantom/stalker as anything in stalkers)
```